src/parsers/js/transformers/babel6/index.js

```diff
@@ -26,7 +26,7 @@ export default {
     );
 
     return babel.transform(code, {
-      presets,
+      presets: [],
```

Comment:
Ah OK. I thought something that worked before wouldn't work anymore, but that doesn't seem to have worked, ever.